src/styles/blocks/page.scss

```diff
@@ -0,0 +1,16 @@
+.page {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+h1,
+h2,
+p,
+.page__body {
+  margin: 0;
+}
+
+button {
+  border: none;
+  outline: none;
+  padding: 0;
+}
```

Comment:
why you reseting your styles in this file? would be better to create separate files for this and don't combine styles for bem blocks / elements and reseting default styles

Reply:
It was a little file, so I desided to add sense into) But, you're right)